webapp/components/experiments/ExperimentDialog.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog/Dialog';
+import Button from '@material-ui/core/Button';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+const useStyles = () => ({
+  root: {
+    '& .cancel': { marginRight: 10 },
+    '&. heading': { color: 'white' },
+  },
+});
+
+const ExperimentDialog = (props) => {
+  const {
+    classes,
+    open,
+    dialogClose,
+    createExperiment,
+  } = props;
+
+  return (
+    <Dialog
+      fullWidth
+      maxWidth="sm"
+      open={open}
+      onClose={() => dialogClose(false)}
+    >
+      <DialogTitle>Create New Experiment</DialogTitle>
+      <DialogContent>
+        <Typography variant="subtitle" className={classes.heading}>The new experiment will replace the current experiment in design, are you sure you want to proceed?</Typography>
+      </DialogContent>
+      <DialogActions>
+        <Button
+          onClick={() => dialogClose(false)}
+          className={classes.cancel}
+          key="CANCEL"
+        >
+          CANCEL
```

Comment:
This is a good example why hardcoded strings lead to inconsistency, we already use "Cancel" in lowercase for other dialogs, now we introduce "CANCEL".

We should have one place where we define what the label for cancel button is and reuse that across all dialogs